Automattic/wp-calypso
# Pricing Grid: Show Free and Enterprise in the main grid

#### Proposed Changes

* This PR is one among several stacked PRs that implements the new pricing grid design. In https://github.com/Automattic/wp-calypso/pull/71372, we introduced a new signup flow /start/onboarding-2023-pricing-grid that is only available in dev and wpcalypso environments.
* The design for the pricing grid can be found in 2e991-pb. Please keep the Figma design open in a separate tab.
* The parent PR is https://github.com/Automattic/wp-calypso/pull/71378.
* This PR brings the Free plan and Enterprise plan into the main pricing grid. 

<img width="1433" alt="Screenshot 2022-12-26 at 11 52 15 AM" src="https://user-images.githubusercontent.com/1269602/209511291-d028dbb9-f14b-4075-9cbd-123d5b9b36a0.png">

* I know what you're thinking - _the UI is ugly and looks nothing like the design_. Correct, and it is not the objective of this PR to introduce any UI changes. It simply adds the necessary below-the-hood changes that enables the free plan and enterprise plan to show in the main pricing grid.

#### What Not To Test

* Responsive screens. It will be implemented in a separate PR. Right now, test in a full width desktop browser viewport.
* Non-en. The first launch will be `en` only.
* UI conformity with Figma design. There are no UI changes in this PR.

#### Known Issues

* Clicking the CTAs on the Free and Enterprise plan don't work and are fixed in a subsequent PR. 

#### Testing Instructions



* Code-wise, verify that the free and enterprise plans attributes are added correctly.
* Verify that the Free and Enterprise plan show in the main pricing grid as shown in the screenshot above.
